UPGRADE-3.0.md

```diff
@@ -1106,4 +1106,15 @@ UPGRADE FROM 2.x to 3.0
 
 ### HttpFoundation
 
-* `Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface` no longer implements the `IteratorAggregate` interface. Use the `all()` method instead of iterating over the flash bag.
+ * `Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface` no longer implements the `IteratorAggregate` interface. Use the `all()` method instead of iterating over the flash bag.
+ * The `PdoSessionHandler` to store sessions in a database changed significantly.
+   The following changes must be made to your `session` table:
+
+   - Rename the `sess_time` to `sess_expiry`. In MySQL this would be:
+      ```
+      ALTER TABLE `session` CHANGE `sess_time` `sess_expiry` INTEGER UNSIGNED NOT NULL;
```

Comment:
Yes, maybe the steps must be:

 - Create a new `sess_expiry` column.
 - Migrate the old data: `UPDATE session SET sess_expiry = sess_time + sess_lifetime;`
 - Remove the `sess_time` and `sess_lifetime` columns.

What do you think?